blocked-edges/4.14.0-ARODNSWrongBootSequence.yaml

```diff
@@ -0,0 +1,15 @@
+to: 4.14.0
+from: .*
+url: https://issues.redhat.com/browse/OCPBUGS-35300
+name: ARODNSWrongBootSequence
+message: |-
+  Disconnected clusters or clusters with a UDR 0.0.0.0/0 route definition that are blocking the ARO ACR and quay, are not be able to add or replace nodes after an upgrade
+matchingRules:
+- type: PromQL
+  promql:
+    promql: |
+      (
+        group(cluster_operator_conditions{name="aro"})
```

Comment:
nit: we recommend [including `_id=""`][1] to future-proof for HyperShift/hosted, see #3591).  Although HyperShift ARO isn't a thing today, so you can skip that if you want.

[1]: https://github.com/openshift/cincinnati-graph-data/blame/ed47a1518646381ff54fdac170d51ecdeed17274/README.md#L190-L192

Reply:
I'll go ahead and skip for now but I'll note that for the future. Thanks.